README.md

```diff
@@ -1 +1,53 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project was created as part of the Code Reviewers training job application at Microverse.org
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML/CSS
+
+## Live Demo
+
+[Live Demo Link](https://htmlpreview.github.io/?https://github.com/SarvarKh/Hello-Microverse/blob/setup/index.html)
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+    Step 1: git clone https://github.com/SarvarKh/to-do-list.git
```

Comment:
- [ ] Please modify the repository link in step 1 , make sure you use the correct link.